.github/workflows/tag.yml

```diff
@@ -43,9 +43,18 @@ jobs:
           current_version: ${{ steps.get-latest-tag.outputs.tag }}
           level: ${{ steps.release-label.outputs.level }}
 
+      - name: Get commit ID for the new tag
+        id: get-commit-id
+        run: echo "commit_id=$(git rev-parse HEAD)" >> $GITHUB_ENV
+
+      - name: Extract PR number from commit message
```

Comment:
Just by looking at the old workflow,  why not use the `github.event.pull_request.number` variable instead. This step feels redundant and unstable

Reply:
Because that didn't work. That's the whole point of this PR to fix this

Reply:
Ah, my bad. What about this environment variable instead:
```
steps.pr.outputs.number
```
We have the `actions-ecosystem/action-get-merged-pull-request@v1.0.1` workflow, which should already do this for us